### ISSUE TITLE
Prevent drop event to occur when draggable item is dropped into input type="text".

### ISSUE THREAD

**Issue by mtadd** [marceljuenemann/angular-drag-and-drop-lists#185]
Currently, when I drag a draggable item over an input of type="text", the input is focussed, and if I drop it there, it sets the input text value to the JSON of the draggable item.   

Is there a way to prevent input elements to accept the draggable item?

**Comment by copooper** [comment]
Putting ondrop="return false;" on input elements will be ok

**Comment by marceljuenemann** [comment]
This is the result of using native drag-and-drop and the fact that IE only supports text as mime type. So you can drag elements to whatever place you like, even to other non-browser applications. Now I'm thinking I should create another version of the directive that doesn't need these quirks by dropping IE support...

**Comment by moogleberry** [comment]
This is exactly what I was looking for.  If there's no other solution, I think either a mention of this in the docs would be great, or a companion directive that adds that to the element (though that might be overkill for a one-line solution).

**Comment by marceljuenemann** [comment]
I have a fix in the [dev branch](https://github.com/marceljuenemann/angular-drag-and-drop-lists/tree/dev) that fixes this problem for all browsers other than IE. Use the dev branch "on your own risk", I'll eventually publish it as version 2.0 when I got a few other things in and tested it thoroughly.

If anyone is interested in the details of the problem, and how I'm planning to work around it, I wrote a [design document](https://github.com/marceljuenemann/angular-drag-and-drop-lists/wiki/Data-Transfer-Design) on it.
